### PR TITLE
Clean up proto naming

### PR DESCRIPTION
**Pull request by sbezverk**
Closes: #172 #173 #174 

* Don't use unsigned integers in proto
* Change “ID” to “Id” in Method names
* Every “noun” should have a message, so Volume should be a message

These are breaking changes from the 0.1 release.

### CODE REVIEW

File: csi.proto [hunk 1]
```diff
@@ -413,7 +413,7 @@ message ListVolumesRequest {
 
 message ListVolumesResponse {
   message Entry {
-    VolumeInfo volume_info = 1;
+    Volume volume_info = 1;
```
Comment (saad-ali) [inline csi.proto]: `volume_info` -> `volume`?

**Comment by saad-ali** [comment]
Minor change requested, otherwise, LGTM
Thanks @sbezverk

**Comment by sbezverk** [comment]
@saad-ali thanks for catching that, fixed.

**Comment by sbezverk** [comment]
@jieyu please review when you have a chance

File: spec.md [hunk 2]
```diff
@@ -717,7 +717,7 @@ message ControllerPublishVolumeResponse {
   // The SP specific information that will be passed to the Plugin in
   // the subsequent `NodePublishVolume` call for the given volume.
   // This information is opaque to the CO. This field is OPTIONAL.
-  map<string, string> publish_volume_info = 1;
+  map<string, string> publish_volume = 1;
```
Comment (jieyu) [inline spec.md]: I would actually rename this to `publish_info`. `publish_volume` sounds a bit weird.

Comment (sbezverk) [reply spec.md]: Yep, +1 for publish_info, done

**Comment by sbezverk** [comment]
Closing as the branch stoped accepting any changes. This PR addresses the same issues: https://github.com/container-storage-interface/spec/pull/182

**Pull request closed without merging**